x-pack/plugins/infra/public/services/telemetry/types.ts

```diff
@@ -61,6 +61,11 @@ export type InfraTelemetryEventParams =
   | HostsViewQueryHostsCountRetrievedParams
   | AssetDetailsFlyoutViewedParams;
 
+export interface PerformanceMetricInnerEvents {
+  key1?: string;
+  value1?: number;
```

Comment:
This looks a bit odd? What's with the 1 in the property names?

Reply:
This is how the event schema defines extra keys https://github.com/elastic/kibana/blob/main/dev_docs/tutorials/performance/adding_custom_performance_metrics.mdx#L68